packages/ckit/src/__tests__/secp256k1-sudt.spec.ts

```diff
@@ -343,8 +343,7 @@ test('test serialize and deserialized', async () => {
   const serialized = builder.serialize(unsigned);
   const deserialized = TransferCkbBuilder.serde.deserialize(serialized);
 
-  const txHash = await provider.sendTransaction(await genesisSigner.seal(deserialized));
-
+  const txHash = await provider.sendTxUntilCommitted(await genesisSigner.seal(deserialized));
```

Comment:
why we need to change it to sendUntilCommitted

Reply:
This line is the last line in the test case here, I think if we don't wait until commited here, the next test case may take some cell that already been used in this test case, thus causing next test case to fail with a time out error.

Reply:
i see, make sense, and this is probably the reason why ckit always failed in CI before